doc/index.rst

```diff
@@ -0,0 +1,73 @@
+Welcome to php-opencloud!
+=========================
+
+Installation
+------------
+
+You must install this library through Composer:
+
+.. code-block:: bash
+
+  composer require rackspace/php-opencloud
+
+
+If you do not have Composer installed, please consult the `official docs
+<https://getcomposer.org/doc/00-intro.md>`_.
+
+Once you have installed the library, you will need to load Composer's autoloader
+(which registers all the required namespaces). To do this, place the following
+line of PHP code at the top of your application's PHP files:
+
+.. code-block:: php
+
+  require 'vendor/autoload.php';
+
+This assumes your application's PHP files are located in the same folder as
+``vendor/``. If your files are located elsewhere, please supply the path to
+``vendor/autoload.php`` in the require statement above.
+
+Read the `OpenStack Getting Started guide <getting-started-with-openstack.html>`_
```

Comment:
Should the links on this line and the next end with `.html`? Asking because the `users` link on https://github.com/rackspace/php-opencloud/pull/549/files#diff-00538670582db41d732b5d707262f15aR79 doesn't end with `.html`.